### DISCUSSION TITLE
Why did you choose a JS-based scrollbar instead of the native one?

### DISCUSSION THREAD

**Discussion by dolphin-wood** [idiotWu/smooth-scrollbar#317] [General]
Hi folks! This question has confused me for years.

JavaScript based scrollbar like this one mimics native scrolling by interpolating scrolling deltas. However, as is explained in [caveats](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/caveats.md#native-behavior-is-unmatchable), the native scrolling behavior is unmatchable and plugins like this will make users uncomfortable especially on mobile devices. So why did you choose the smooth-scrollbar instead of native scrollbar based solutions like [simplebar](https://github.com/Grsmto/simplebar) or [OverlayScrollbars](https://github.com/KingSora/OverlayScrollbars)?

FYI: [scrollbar plugin comparison table](https://kingsora.github.io/OverlayScrollbars/#!faq)

**Comment by sadeghbarati** [comment]
Smooth :sunglasses:   > Normal :sleepy: 

only for some cases and consider about **mobile** devices and **big screens**

Thanks for the plugin (joy of scroll)❤️

**Comment by eleweek** [comment]
I needed a smooth scroll bar in an area with code. I have a text area with code and I want to smoothly scroll to an executed line. Example: https://just-taking-a-ride.com/inside_python_dict/chapter3.html

Thanks for the plugin!

**Comment by blitzve0** [comment]
Its Smooooootttthhhhh Like Butter when you set damping to 0.02 👅💦😛😋
